test/class-name-generator.html

```diff
@@ -0,0 +1,116 @@
+<!doctype html>
+
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../iron-test-helpers/mock-interactions.js"></script>
+
+  <link rel="import" href="helpers.html">
+  <link rel="import" href="../all-imports.html">
+</head>
+
+<body>
+
+  <script>
+    describe('class name generator', () => {
+      let grid, firstCell, initialCellClasses;
+
+      beforeEach(() => {
+
+        grid = document.createElement('vaadin-grid');
```

Comment:
Would still use text-fixture for now. The whole "no-fixture initiative" is in an early stage and the current take is that we could just replace the import from all of our tests to use Vaadin's test fixture.

Reply:
Done